airgun/browser.py

```diff
@@ -178,6 +179,8 @@ def _get_selenium_browser(self):
             prefs = {'download.prompt_for_download': False}
             options.add_experimental_option("prefs", prefs)
             options.add_argument('disable-web-security')
+            if headless == "true":
```

Comment:
Just `if headless:`.

First, we will obtain this value from Robottelo, where it is casted to boolean. So it will never be equal string "true".
Second, string that contains value that seems to be of other type is good indicator that we should just change variable type.